falcon/testing/srmock.py

```diff
@@ -21,7 +21,17 @@
 from falcon import util
 
 
-class StartResponseMock(object):
+class ResponseMeta(object):
+    @property
+    def status(self):
```

Comment:
It seems refactoring these `status` properties breaks some existing unit tests, @tribals could you please take a look at Travis output?

Reply:
I ran full test suite in both `py27` and `py36` envs before submitting pull request. I fixed all (as I thought) tests broken by this change.

I will take a look at Travis, of course.